src/components/custom-tree-item.jsx

```diff
@@ -41,9 +72,12 @@ const CustomContent = React.forwardRef(function CustomContent(props, ref) {
                 expanded && styles.expanded,
                 selected && styles.selected,
                 focused && styles.focused,
-                disabled && styles.disabled
+                disabled && styles.disabled,
+                hover && { backgroundColor: `${theme.aggrid.highlightColor} !important`, borderRadius: '16px' } // Add hover style with  background color
             )}
             onMouseDown={handleMouseDown}
+            onMouseEnter={() => toggleHover(true)}
```

Comment:
toogle doesn't take any argument. It's mean to "toggle" then it reverse le current boolean value.
Here if you whant to set the boolean value, then it's a set that's all.